ontology/uco/tool/tool.ttl

```diff
@@ -54,139 +60,54 @@ tool:BuildFacet
 tool:BuildInformationType
 	a
 		owl:Class ,
+		owl:DeprecatedClass ,
 		sh:NodeShape
 		;
 	rdfs:subClassOf core:UcoInherentCharacterizationThing ;
 	rdfs:label "BuildInformationType"@en ;
 	rdfs:comment "A build information type is a grouping of characteristics that describe how a particular version of software was converted from source code to executable code."@en ;
-	sh:property
-		[
-			sh:class configuration:Configuration ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:IRI ;
-			sh:path tool:buildConfiguration ;
-		] ,
-		[
-			sh:class tool:BuildUtilityType ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:IRI ;
-			sh:path tool:buildUtility ;
-		] ,
-		[
-			sh:class tool:CompilerType ;
-			sh:nodeKind sh:IRI ;
-			sh:path tool:compilers ;
-		] ,
-		[
-			sh:class tool:LibraryType ;
-			sh:nodeKind sh:IRI ;
-			sh:path tool:libraries ;
-		] ,
-		[
-			sh:datatype xsd:dateTime ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:compilationDate ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildID ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildLabel ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildOutputLog ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildProject ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildScript ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildVersion ;
-		]
-		;
+	rdfs:seeAlso observable:SoftwareBuildFacet ;
+	sh:message "uco-tool:BuildInformationType is deprecated.  Please migrate usage of this class to uco-observable:SoftwareBuildFacet."@en ;
+	sh:not [
+		a sh:NodeShape ;
+		sh:class tool:BuildInformationType ;
+	] ;
 	sh:targetClass tool:BuildInformationType ;
 	.
 
 tool:BuildUtilityType
 	a
 		owl:Class ,
+		owl:DeprecatedClass ,
 		sh:NodeShape
 		;
 	rdfs:subClassOf core:UcoInherentCharacterizationThing ;
 	rdfs:label "BuildUtilityType"@en ;
 	rdfs:comment "A build utility type characterizes the tool used to convert from source code to executable code for a particular version of software."@en ;
-	sh:property
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildUtilityName ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:cpeid ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:swid ;
-		]
-		;
+	rdfs:seeAlso observable:BuildUtilityFacet ;
```

Comment:
`BuildUtilityFacet` was not described directly in the proposal, but was referenced in some of the migration notes.

@sbarnum - Do you have an `rdfs:comment` for this class?